### ISSUE TITLE
Quokka for Intellij WebStorm - ActionUpdateThread.OLD_EDT` is deprecated

### ISSUE THREAD

**Issue by rcoundon** [wallabyjs/quokka#947]
### Issue description or question

Is this issue related to Quokka not outputting the expected results of your code?: No

When starting WebStorm the following error is displayed:

```
com.intellij.diagnostic.PluginException: `ActionUpdateThread.OLD_EDT` is deprecated and going to be removed soon. 'com.quokka.javascript.actions.ProfileQuokkaFile' must override `getActionUpdateThread` and chose EDT or BGT. See ActionUpdateThread javadoc. [Plugin: quokka.js]
	at com.intellij.diagnostic.PluginProblemReporterImpl.createPluginExceptionByClass(PluginProblemReporterImpl.java:23)
	at com.intellij.diagnostic.PluginException.createByClass(PluginException.java:90)
	at com.intellij.diagnostic.PluginException.reportDeprecatedUsage(PluginException.java:125)
	at com.intellij.openapi.actionSystem.ActionUpdateThreadAware.getActionUpdateThread(ActionUpdateThreadAware.java:21)
	at com.intellij.openapi.actionSystem.AnAction.getActionUpdateThread(AnAction.java:199)

This doesn't affect the running of the plugin but I imagine it will when the deprecated capabilities are removed in the future

### Sample code
N/A

### Sample repository link
N/A

### Quokka.js Console Output
N/A

### Code editor version
Webstorm 2024.1

### OS name and version
Mac Sonoma 14.4.1

**Comment by mburnell** [comment]
Thanks for reporting this issue. We're aware of it, and have held off on addressing it in order to maintain compatibility with older JetBrains editors. Please see https://github.com/wallabyjs/public/issues/3327#issuecomment-1926101821 for additional information. We understand this has become more of an annoyance in recent versions, and will see what we can do to fix or suppress this warning.

**Comment by mburnell** [comment]
This issue has been addressed in version `1.0.444`; after updating to this version of the Quokka plugin, this warning/error should no longer occur.
